server/src/main/java/org/elasticsearch/transport/TransportStats.java

```diff
@@ -42,6 +57,14 @@ public TransportStats(StreamInput in) throws IOException {
         rxSize = in.readVLong();
         txCount = in.readVLong();
         txSize = in.readVLong();
+        if (in.getVersion().onOrAfter(Version.V_8_1_0)) {
+            handlingTimeBucketFrequencies = in.readLongArray();
+            handlingTimeBucketBounds = in.readIntArray();
```

Comment:
Why write these over the wire when they're hard coded per version anyway? :) I guess we need it to make BwC safe?